contracts/farm/interfaces/IMasterChefSushi.sol

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
I am trying to move away from having a miriad of interfaces. please move this to contracts/interface folder lets have all the interfaces there